packages/xo-server/src/xapi/index.mjs

```diff
@@ -46,8 +46,10 @@ import {
 const log = createLogger('xo:xapi')
 
 class AggregateError extends Error {
-  constructor(errors, message) {
+  constructor(errors, { code, message, params }) {
     super(message)
+    this.code = code
+    this.params = params
     this.errors = errors
   }
```

Comment:
In theory you did the right thing, but `AggregateError` is a recent standard JavaScript error, this class is polyfill for it (until we use a recent version of Node to have it natively) and we should not change the API.

For our use case, I think we should simply assign the properties to the instance.